src/com/lftechnology/java/training/krishna/collections/generics/MoterBike.java

```diff
@@ -0,0 +1,51 @@
+
+package com.lftechnology.java.training.krishna.collections.generics;
+
+/**
+ * This program store moterbike information
+ * 
+ * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+ * @version 1.0
+ */
+public class MoterBike extends Vehicle {
+
+	private int gears;
+	private int CC;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "CC" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)